src/components/Header/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+const Header = () => {
+  return (
+    <>
+      <div class="navigation-container flex-hz">
+        <div class="navigation-item-1">
+          <h5 class="lib-name-1">NaariCare <span class="lib-name-2">Quiz</span></h5>
+        </div>
+        <div class="navigation-item-3 flex-hz">
+          <Link className="category-link" to="/"> <button class="btn btn-primary" >Home</button></Link>
```

Comment:
Since the link is rendered as an anchor tag, why not use the button classes in the <Link /> and render it without adding a button child within <Link>?